charts/images.yaml

```diff
@@ -73,6 +73,80 @@ imagesForVersion:
     fluentd:
       repository: 'keppel.$REGION.cloud.sap/ccloud/kubernikus-fluentd'
       tag: 'v1.14.6-1.1'
+  '1.22.12':
+    supported: true
+    apiserver:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kube-apiserver'
+      tag: 'v1.22.12'
+    controllerManager:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kube-controller-manager'
+      tag: 'v1.22.12'
+    scheduler:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kube-scheduler'
+      tag: 'v1.22.12'
+    etcd:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-dockerhub-mirror/sapcc/etcd'
+      tag: 'v3.3.14'
+    etcdBackup:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-dockerhub-mirror/sapcc/etcdbrctl'
+      tag: '0.5.2'
+    kubelet:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kubelet'
+      tag: 'v1.22.12'
+    kubeProxy:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kube-proxy'
+      tag: 'v1.22.12'
+    cloudControllerManager:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/openstack-cloud-controller-manager'
+      tag: 'v1.22.0'
+    dex:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/dex'
+      tag: '38f4f8ea8d487470a1dd5b83d66b428d8b502f81'
+    dashboardProxy:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/keycloak-gatekeeper'
+      tag: '6.0.1'
+    dashboard:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-dockerhub-mirror/kubernetesui/dashboard'
+      tag: 'v2.0.4'
+    coreDNS:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-dockerhub-mirror/sapcc/coredns'
+      tag: '1.6.2'
+    pause:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-dockerhub-mirror/sapcc/pause-amd64'
+      tag: '3.1'
+    wormhole:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kubernikus'
+      tag: 'changeme' #this is injected to match the operator so far
+    csiAttacher:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-attacher'
+      tag: 'v3.3.0'
+    csiProvisioner:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-provisioner'
+      tag: 'v3.0.0'
+    csiSnapshotter:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-snapshotter'
+      tag: 'v4.2.1'
+    csiSnapshotController:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-snapshot-controller'
+      tag: 'v4.2.1'
+    csiResizer:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-resizer'
+      tag: 'v1.3.0'
+    csiLivenessProbe:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-livenessprobe'
+      tag: 'v2.4.0'
+    csiNodeDriver:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/csi-node-driver-registrar'
+      tag: 'v2.3.0'
+    cinderCSIPlugin:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/cinder-csi-plugin'
+      tag: 'v1.22.0'
+    flannel:
+      repository: 'keppel.$REGION.cloud.sap/ccloud-quay-mirror/coreos/flannel'
+      tag: 'v0.12.0'
+    fluentd:
+      repository: 'keppel.$REGION.cloud.sap/ccloud/kubernikus-fluentd'
+      tag: 'v1.14-1'
   '1.22.4':
     supported: true
```

Comment:
We should set this to `false` to have the version not show um in elektra anymore. Otherwise lgtm